### DISCUSSION TITLE
Bind commands to multiple key combinations

### DISCUSSION THREAD

**Discussion by jeff-bowles** [dbeaver/dbeaver#21091] [Q&A] [answered]
This is a "nice to have" feature.

I would like to be able to assign two different key sequences to a single command.

Currently, I only have one example.  I would like to bind "Find and Replace" to both the default CTRL H and CTRL F.
* CTRL H  This is "Find and Replace" in both Eclipse and Scintilla-based editors
* CTRL F  This is "Find" in Scintilla-based editors and many other software packages

This is an aid to developers with lots of experience in editors other than Eclipse.   I find myself pressing CTRL F often, with no result.  It's a little frustrating.

Jeff Bowles

**Comment by E1izabeth** [comment, marked as answer]
I believe it's possible.
In `Window->Preferences->User Interface->Keys` you can find commands and shortcuts for them.
To assign 2 shortcuts for the same command you could select the desired command and press the "Copy Command" button. This will create a new command to which you can assign a new shortcut. Note, that if the new shortcut is already in use, you need to change the shortcut for the command use it first.

I tried to do it for your case and it works for me. If you need more details, please ask.
